usagov/vote-gov-drupal
# VOTE-1135: Update content manager role permission

## Jira ticket

[VOTE-1135](https://cm-jira.usa.gov/browse/VOTE-1135)

## Description

Allow content managers to publish landing pages.

## Deployment and testing

### Post-deploy steps

none

### QA/Testing instructions

1. Locally run `lando retune`
2. Login as a user with the 'Content Manager' role. `lando drush uli --name=username` will allow you to login as a user with the Content Manager role.
3. Go to the Content admin page, locate a Landing Page content item, click edit, and confirm that you have the publishing option below. Additionally, confirm that you can add a revision log entry too.
4. Confirm the same for a translated version of the page.
5. For other node types, confirm the workflow/moderation states work correctly and that you can add a revision log entry.

## Checklist for the Developer




## Checklist for the Peer Reviewers
